.gitignore

```diff
@@ -20,6 +20,7 @@ Thumbs.db
 .tmproj
 *.sublime-project
 *.sublime-workspace
+*.lnk
```

Comment:
Is this new line related to the history navigation fix?
If it's not, it should be removed from this PR.
(and it would be a good occasion afterwards to show you how to "squash" your modifications into a single commit ;-) )